shopping-cart/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Shopping Cart</title>
+    <link rel="stylesheet" href="../common/main.css">
+    <link rel="stylesheet" href="shopping-cart.css">
+</head>
+<body>
+        <header>
+            <h1>Sure Shot Shopping Cart</h1>
+        </header>
+        <nav>
+            <a href="../index.html">Home</a>
+            <a href='../products/index.html'></a>
+            <a href="../shopping-cart/">Cart</a>
+        </nav>
+</body>
+<table>
+    <thread>
+        <tr>
+            <th>
+            </th>
+        </tr>
+    </thread>
+</table>
+<tbody>
+    <th></th> 
+</th>
+</tbody>
```

Comment:
You want the table tag inside the body tag and the tbody tag inside the table tag